in-app-wallet/typescript/build/in_app_wallet_examples.js

```diff
@@ -0,0 +1,97 @@
+    const [coin1, coin2] = txb.splitCoins(txb.object(sourceCoinId), [txb.pure(10000), txb.pure(20000)]);
+    // Each new object created in a transaction must have an owner
+    txb.transferObjects([coin1, coin2], txb.pure(sender));
+    return txb;
+};
+// The transaction block with gas context 
+const progTxnSplit = (sender, sourceCoinId) => {
+    const txb = new sui_js_1.TransactionBlock();
+    // Split two new coins out of sourceCoinId, one with 10000 balance, and the other with 20000
+    const [coin1, coin2] = txb.splitCoins(txb.object(sourceCoinId), [txb.pure(10000), txb.pure(20000)]);
+    // Each new object created in a transaction must have an owner
+    txb.transferObjects([coin1, coin2], txb.pure(sender));
+    txb.setSender(sender);
+    txb.setGasBudget(GAS_BUDGET);
+    txb.setGasOwner(sender);
+    return txb;
+};
+const inAppWalletE2E = async () => {
+    // Create an ephemeral session token to access In-App Wallet functionality
+    const sessionToken = await keyService.shinami_key_createSession(secret);
+    // Create a new wallet (can only be done once with the same walletId)
+    //const createdWalletAddress = await walletService.shinami_wal_createWallet(walletId, sessionToken);
+    // Retrieve the wallet address via the walletId. Should be the same as createdWalletAddress
+    const walletAddress = await walletService.shinami_wal_getWallet(walletId);
+    console.log("WAL ADDR:", walletAddress);
+    // Deposit some SUI to the wallet via the faucet
+    //const faucetResponse = await suiProvider.requestSuiFromFaucet(walletAddress);
+    //const sourceCoinId = faucetResponse.transferredGasObjects[0].id;
+    const sourceCoinId = "0x8388d2d326495c3da164804612d28c58754fa684164d259794cff70cc09ea815";
+    console.log("SOURCE COIN:", sourceCoinId);
+    // // Get the transaction block of the full transaction.
+    // const txbFull = progTxnSplit(walletAddress, sourceCoinId);
+    // // Generate the bcs serialized transaction payload
+    // const payloadBytesFull = await txbFull.build({ provider: suiProvider });
+    // // Convert the payload byte array to a base64 encoded string
+    // const payloadBytesFullBase64 = btoa(
+    //     payloadBytesFull.reduce((data, byte) => data + String.fromCharCode(byte), '')
+    // );
+    // // Sign the payload via the Shinami Wallet Service.
+    // const sig = await walletService.shinami_wal_signTransactionBlock(walletId, sessionToken, payloadBytesFullBase64);
+    // // Execute the signed transaction on the Sui blockchain
+    // const executeResponseFull = await suiProvider.executeTransactionBlock(
+    //     {
+    //         transactionBlock: payloadBytesFullBase64,
+    //         signature: sig,
+    //         options: {showEffects: true},
+    //         requestType: "WaitForLocalExecution"
+    //     }
+    // );
+    // console.log("Execution Status:", executeResponseFull.effects?.status.status);
```

Comment:
What's the plan for this commented code?

Reply:
ack sorry, not used to the non-auto save of vscode. This should all be cleaned up.

Reply:
Oh this is the compiled js. shouldn't be merged in